osf/migrations/0015_add_identifiers_to_preprints.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2017-04-19 19:38
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.contrib.contenttypes.models import ContentType
+
+from osf.models import PreprintService, Identifier
+from website.identifiers.utils import get_or_create_identifiers
+
+
+def add_identifiers_to_preprints(*args):
+    preprints_without_identifiers = PreprintService.objects.filter(identifiers__isnull=True)
+
+    for preprint in preprints_without_identifiers:
+        new_identifiers = get_or_create_identifiers(preprint)
```

Comment:
Drive-by: I'm not quite sure we want to do this in a Django migration. For one, this is is sending requests to an external service, EZID, which we can't always assume is available. Also, developers will not need this migration to be run in their local environment. This might be better off as a one-off script in scripts/.

Thoughts?